README.md

```diff
@@ -24,13 +24,14 @@ In order to use this library, you must first download it from PyPi: `pip install
 ```py
 import securepy
 
-restrictor = securepy.Restrictor(max_exec_time=3, restriction_scope=2)
+restrictor = securepy.Restrictor(max_exec_time=3, restriction_scope=2, stdin="yourstringhere")
```

Comment:
This adds `stdin` as argument to `securepy.Restrictor`, not to `StdCapture` (now named `IOCage`).
Add this argument to the STDOUT/STDERR Capturing section and document it below the code example.

(Also applies for line 34)